### ISSUE TITLE
Make the windows' Magnifier follows the text insertion point

### ISSUE THREAD

**Issue by renatorroliveira** [microsoft/vscode#43639]
Hi,
I would like to make a request for VS Code team.
The Windows magnifier application has an option to follow the text insertion point, but in the VSCode edito it doesn't work.
In the Visual Studio Community 2017 it works, so I thought it may be possible to support this feature in VSCode as well.

This feature is very useful for people with low vision that relies on screen magnifiers. I am not sure if this is a VSCode issue/feature request or if it is specific of magnifier app. There are other applications where the magnifier cannot follow the text insertion point, like Atom or Sublime Text.

I think it may be limitations on the editor library, but I will appreciate if this is solved for VSCode since I have low vision and use the magnifier all the time =)

- VSCode Version: 1.20
- OS Version: Windows 10 - 1709 - Build16299.214

Steps to Reproduce:

1. Open any text file.
2. Move the text insertion cursor or type anything.

**Comment by egamma** [comment]
This is the mentioned setting.
![image](https://user-images.githubusercontent.com/172399/36257665-1d7af0c8-1258-11e8-8952-745676f97a06.png)

It works in Chrome, that is, the view is scrolled when the cursor moves out of the screen.

**Comment by renatorroliveira** [comment]
Yes, that's right, it moves with the text insertion point, without the need to move it with the mouse every few words typed in. =)

**Comment by isidorn** [comment]
Dup of https://github.com/microsoft/vscode/issues/7394
